.editorconfig

```diff
@@ -0,0 +1,263 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = false
+max_line_length = 160
+tab_width = 4
+ij_continuation_indent_size = 8
```

Comment:
Are you sure that you want code formatted like this:
```java
List<Path> notCompressedFiles = pathStream
        .sorted(oldestFirst)
        .collect(Collectors.toList());
```

or rather like this:
```java
List<Path> notCompressedFiles = pathStream
    .sorted(oldestFirst)
    .collect(Collectors.toList());
```
if you prefer 2nd option then `continuation_indent_size` should be equal `4`

Reply:
8 is good